lib/oban/query.ex

```diff
@@ -84,6 +84,26 @@ defmodule Oban.Query do
     :ok
   end
 
+  @spec retry_all_jobs(Config.t(), Ecto.Queryable.t()) :: :ok
+  def retry_all_jobs(conf, queryable) do
+    query =
+      queryable
+      |> where([j], j.state not in ["available", "executing", "scheduled"])
+      |> update([j],
+        set: [
+          state: "available",
+          max_attempts: fragment("GREATEST(?, ? + 1)", j.max_attempts, j.attempt),
+          scheduled_at: ^utc_now(),
+          completed_at: nil,
+          cancelled_at: nil,
+          discarded_at: nil
+        ]
+      )
+
+    Repo.update_all(conf, query, [])
+    :ok
```

Comment:
Regarding my count comment above, something like this:

```suggestion
    {count, _} = Repo.update_all(conf, query, [])
    
    {:ok, count}
```